navigation/index.tsx

```diff
@@ -3,29 +3,60 @@
  * https://reactnavigation.org/docs/getting-started
  *
  */
-import { Feather, FontAwesome } from '@expo/vector-icons';
-import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
-import { NavigationContainer, DefaultTheme, DarkTheme } from '@react-navigation/native';
-import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import * as React from 'react';
-import { ColorSchemeName, Pressable } from 'react-native';
+import { Feather } from "@expo/vector-icons";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import {
+  NavigationContainer,
+  DefaultTheme,
+  DarkTheme,
+} from "@react-navigation/native";
+import { pressedOpacity } from "../constants/Values";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import * as React from "react";
+import {
+  StyleSheet,
+  ColorSchemeName,
+  View,
+  Text,
+  TouchableOpacity,
+} from "react-native";
 
-import Colors from '../constants/Colors';
-import useColorScheme from '../hooks/useColorScheme';
-import ModalScreen from '../screens/ModalScreen';
-import NotFoundScreen from '../screens/NotFoundScreen';
-import TabOneScreen from '../screens/TabOneScreen';
-import TabTwoScreen from '../screens/TabTwoScreen';
-import { RootStackParamList, RootTabParamList, RootTabScreenProps } from '../types';
-import LinkingConfiguration from './LinkingConfiguration';
-import ProfileScreen from '../screens/ProfileScreen';
-import PostScreen from '../screens/PostScreen';
+import Colors from "../constants/Colors";
+import useColorScheme from "../hooks/useColorScheme";
+import ModalScreen from "../screens/ModalScreen";
+import NotFoundScreen from "../screens/NotFoundScreen";
+import HomeScreen from "../screens/HomeScreen";
+import SavedScreen from "../screens/SavedScreen";
+import ChatScreen from "../screens/ChatScreen";
+import ProfileScreen from "../screens/ProfileScreen";
 
-export default function Navigation({ colorScheme }: { colorScheme: ColorSchemeName }) {
+import {
+  RootStackParamList,
+  RootTabParamList,
+  RootTabScreenProps,
+} from "../types";
+import LinkingConfiguration from "./LinkingConfiguration";
+import PostScreen from "../screens/PostScreen";
+import Home from "../assets/svg-components/home";
+import Header from "../assets/svg-components/header";
+import ClickedHome from "../assets/svg-components/home_clicked";
+import BookMark from "../assets/svg-components/bookmark";
+import ClickedBookMark from "../assets/svg-components/bookmark_clicked";
+import ClickedChat from "../assets/svg-components/clicked_chat";
+import Chat from "../assets/svg-components/chat";
+import ClickedProfile from "../assets/svg-components/clicked_profile";
+import Profile from "../assets/svg-components/profile";
```

Comment:
woah this is too many imports for `svg-components`. Maybe we should create an `index.ts` in `svg-components` that serves as a directory for all svgs so that we can do `import { Home, Header, ...} from "../assets/svg-components/index"`